src/BuyLotto.js

```diff
@@ -0,0 +1,178 @@
+const { Random, Console } = require("@woowacourse/mission-utils");
+const {
+  MESSAGE,
+  CONDITION,
+  PRIZE_MONEY,
+  ERR_MESSAGE,
+} = require("./constant/constant");
+const validate = require("./Validate");
+
+class BuyLotto {
+  constructor(
+    howMany,
+    makeNumbers,
+    userInputNum,
+    userInputBonusNum,
+    fifthPrize,
+    fourthPrize,
+    thirdPrize,
+    secondPrize,
+    firstPrize,
+    sum,
+    getYield
+  ) {
+    this.howMany = howMany;
+    this.makeNumbers = makeNumbers;
+    this.userInputNum = userInputNum;
+    this.userInputBonusNum = userInputBonusNum;
+    this.fifthPrize = fifthPrize;
+    this.fourthPrize = fourthPrize;
+    this.thirdPrize = thirdPrize;
+    this.secondPrize = secondPrize;
+    this.firstPrize = firstPrize;
+    this.sum = sum;
+    this.getYield = getYield;
+  }
```

Comment:
매개변수가 너무 많은 것 같아용! BuyLotto 인스턴스를 호출할 때도 인자 값으로 들어가는게 없으니 
constructor에서 초기화하는게 어떨까요? 그리고 prize는 Map 객체나 배열로 관리하는게 좋을 것 같습니다. 방법은 다양하게 있을 것 같아요!😊

```suggestion
  constructor() {
    this.sum = 0;
    this.getYield = 0;
    this.howMany = 0;
    this.makeNumbers = [];
    this.userInputNum = [];
    this.userInputBonusNum = 0;
    this.prize = new Map([
      ['fifthPrize', 0],
      ['fourthPrize', 0],
      ['thirdPrize', 0],
      ['secondPrize', 0],
      ['firstPrize', 0],
    ]);  
  }
```

Reply:
저도 만들고보니 인자가 너무 많은거같아서 고민을 하고있었는데 좋은 의견주셔서 감사합니다! 참고해서 공부해보도록 하겠습니다